meinberlin/static/embed.js

```diff
@@ -0,0 +1,50 @@
+/* global $ */
+$(document).ready(function () {
+  var $main = $('main')
+
+  var loadHtml = function (html) {
+    var $root = $(html).filter('main')
+    $main.empty()
+    $main.append($root.children())
+    onReady()
+  }
+
+  var onReady = function () {
+    // adhocracy4.onReady($target)
+
+    $main.find('form[action]').submit(function (event) {
+      event.preventDefault()
+      var form = this
+
+      // timeout required for use with CKEditor
```

Comment:
It seems to do bind to the submit event, too. So we have to ensure to get the form data *after* ckeditor is done.